src/Components/Facility/FacilityHome.tsx

```diff
@@ -75,6 +77,27 @@ export const FacilityHome = ({ facilityId }: Props) => {
     },
   });
 
+  const fetchImage = async () => {
+    try {
+      let imageUrl = facilityData?.read_cover_image_url;
+      if (imageUrl) {
+        await fetch(`${imageUrl}`, {
+          headers: { "Force-Revalidate": "1" },
+          credentials: "include",
+          mode: "no-cors",
```

Comment:
are these headers required? how does this differ from with and without these?

Reply:
credentials isn't required, mode is though; was getting 502 error for the fetch call without it.